src/styles/_common.scss

```diff
@@ -17,6 +17,7 @@
 @import './foundation/shadows';
 @import './foundation/typography';
 @import './foundation/z-index';
+@import './foundation/focus-ring';
```

Comment:
Is this the right spot?

Reply:
Kiiinda. `_common.scss` is only used privately so this isn't exposing anything new, but I think it's confusing to add a file to the foundations folder and it not be referenced by `foundation.scss` (which is an entry point for the public API).

Can you move the `focus-ring.scss` file into the common folder? That'd help make it clearer that it is a new private file distinct from the public api files (foundation and shared)

Reply:
Yeah, sorry, in the initial PR, I added this to a new `src/styles/common` directory. But it sounds like that won't work from Ben's comment.